pages/studentAssessmentExam/studentAssessmentExam.js

```diff
@@ -55,6 +76,33 @@ router.post('/', function (req, res, next) {
         res.redirect(res.locals.urlPrefix + '/assessment_instance/' + assessment_instance_id);
       }
     );
+  } else if (req.body.__action === 'join_group') {
+    const joinCode = req.body.join_code;
+    groupAssessmentHelper.joinGroup(joinCode, res, function (err, joinErr) {
```

Comment:
Nit:
```suggestion
    groupAssessmentHelper.joinGroup(req.body.join_code, res, function (err, joinErr) {
```

Reply:
Fixed this and caught 3 more of the same one use variable in [0a29279](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/0a29279f6f6e59b9684de38e47475cc05fbf6335)